wemake_python_styleguide/violations/consistency.py

```diff
@@ -1938,3 +1940,34 @@ class MultilineLoopViolation(ASTViolation):
 
     error_template = 'Forbids multiline loops'
     code = 352
+
+
+@final
+class UselessBlankLineViolation(TokenizeViolation):
+    """
+    Forbids useless blank lines in between brackets.
+
+    Reasoning:
```

Comment:
We are missing a solution here: `Remove blank lines from the start and from the end of a collection`